src/org/traccar/protocol/SuntechProtocolEncoder.java

```diff
@@ -25,6 +25,13 @@
     protected Object encodeCommand(Command command) {
 
         switch (command.getType()) {
+            case Command.TYPE_REBOOT_DEVICE:
+                return formatCommand(command, "SA200CMD;{%s};02;Reboot\r", Command.KEY_UNIQUE_ID);
+            case Command.TYPE_POSITION_SINGLE:
+                return formatCommand(command, "SA200GTR;{%s};02;\r", Command.KEY_UNIQUE_ID);
+            case Command.TYPE_OUTPUT_CONTROL:
+                return formatCommand(command, "SA200CMD;{%s};0{%s};Enable{%s}\r",
+                        Command.KEY_UNIQUE_ID, Command.KEY_DATA, Command.KEY_DATA);
```

Comment:
I am not sure, I was expecting it is some params sent by the interface.
I sow a similar usage on Pt502ProtocolEncoder.